scripts/ci/collect-service-logs.sh

```diff
@@ -21,6 +21,19 @@ usage() {
     echo "e.g. ./scripts/ci/collect-service-logs.sh stackrox"
 }
 
+dump_logs() {
+    for ctr in $(kubectl -n "${namespace}" get "${object}" "${item}" -o jsonpath="{${jpath}[*].name}"); do
+        kubectl -n "${namespace}" logs "${object}/${item}" -c "${ctr}" > "${log_dir}/${object}/${item}-${ctr}.log"
+        prev_log_file="${log_dir}/${object}/${item}-${ctr}-previous.log"
+        if kubectl -n "${namespace}" logs "${object}/${item}" -p -c "${ctr}" > "${prev_log_file}"; then
+            exit_code="$(kubectl -n "${namespace}" get "${object}/${item}" -o jsonpath="{${jpath}}" | jq --arg ctr "$ctr" '.[] | select(.name == $ctr) | .lastState.terminated.exitCode')"
+            if [ "$exit_code" -eq "0" ]; then
+            mv "${prev_log_file}" "${log_dir}/${object}/${item}-${ctr}-prev-success.log"
```

Comment:
```suggestion
                mv "${prev_log_file}" "${log_dir}/${object}/${item}-${ctr}-prev-success.log"
```

Reply:
oh wierd, wonder why only that 'one line' didn't copy right, will fix.